libs/@guardian/identity-auth-frontend/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { IdentityAuth } from '@guardian/identity-auth';
+import type { CustomClaims } from '@guardian/identity-auth';
+
+type FrontendIdTokenClaims = CustomClaims & {
+	email: string;
+	braze_uuid: string;
+};
+
+export type FrontendIdentityAuth = IdentityAuth<never, FrontendIdTokenClaims>;
+
+const getStage = (isDev: boolean, stage: string) => (isDev ? 'DEV' : stage);
+
+const getIssuer = (stage: string) =>
+	stage === 'PROD'
+		? 'https://profile.theguardian.com/oauth2/aus3xgj525jYQRowl417'
+		: 'https://profile.code.dev-theguardian.com/oauth2/aus3v9gla95Toj0EE0x7';
+
+const getClientId = (stage: string) =>
+	stage === 'PROD' ? '0oa79m1fmgzrtaHc1417' : '0oa53x6k5wGYXOGzm0x7';
```

Comment:
What are these hardcoded (seemingly random) values? 

Are they specific only to the use on `www.theguardian.com` & if so should we be explicit about that in a comment here & perhaps in a documentation section titled `Can I use this on another domain` or something?

Reply:
They are specifically for use on `www.theguardian.com`. Will add some documentation!